lib/Lower/PFTBuilder.cpp

```diff
@@ -624,6 +934,18 @@ pft::ModuleLikeUnit::ModuleStatement getModuleStmt(const T &mod) {
 
 } // namespace
 
+llvm::cl::opt<bool> ClDisableStructuredFir(
```

Comment:
We should use lowercase here. (I had copied some options from a file that used the old variables must be uppercased rule, but MLIR uses lowercase variables.) My bad.

Reply:
ClDisableStructuredFir --> clDisableStructuredFir